package.json

```diff
@@ -59,6 +60,7 @@
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
     "@chakra-ui/cli": "^2.4.1",
     "@formatjs/cli": "^6.2.7",
+    "@playwright/test": "1.33.0",
```

Comment:
The latest version of playwright should be `1.42.1`, but because the node version is v20.6.1, we get an error when running the tests (see [issue](https://github.com/microsoft/playwright/issues/29253)). So I have forced the v1.33.0. This version is high enough for the vite to be able to run as a webserver without [this issue](https://github.com/microsoft/playwright/issues/21227), but low enough to overcome the node issue